src/app/containers/Article/index.jsx

```diff
@@ -11,46 +11,70 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
+const Container = ({ children, service }) => (
```

Comment:
I can't pinned down exactly why but it feels like `Container` could be better named? 
`ArticleWrapper`, `ArticleFurniture` ....?